indico/modules/events/templates/display/indico/_details.html

```diff
@@ -2,6 +2,8 @@
                                                               render_description %}
 {% from 'attachments/_display.html' import render_attachments_folders %}
 
+## Do not forget to update the conditions in the parent template when you add a new row in the details table.
```

Comment:
I'd rather use `{# ... #}` - we don't use line statements anywhere else in Jinja